lib/validators/project.js

```diff
@@ -52,7 +52,8 @@ class Project extends Validator {
 
       isMergeable = issuesPrCloses.some(issue => idsFromCards.includes(issue))
     }
-    return consolidateResult([constructOutput(validatorContext, projects, validationSettings, {
+    console.log('projects', projects)
```

Comment:
I think you intended for this to be removed. Yes?

Reply:
i'll remove it when i get home later today and merge the PR